californium-core/src/main/java/org/eclipse/californium/core/network/stack/CoapUdpStack.java

```diff
@@ -18,82 +18,74 @@
  *    Kai Hudalla - logging
  *    Kai Hudalla (Bosch Software Innovations GmbH) - use Logger's message formatting instead of
  *                                                    explicit String concatenation
+ *    Achim Kraus (Bosch Software Innovations GmbH) - move common function to BaseCoapStack
  ******************************************************************************/
 package org.eclipse.californium.core.network.stack;
 
-import java.util.List;
-import java.util.concurrent.ScheduledExecutorService;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import org.eclipse.californium.core.coap.EmptyMessage;
-import org.eclipse.californium.core.coap.Request;
-import org.eclipse.californium.core.coap.Response;
-import org.eclipse.californium.core.network.Exchange;
 import org.eclipse.californium.core.network.Outbox;
-import org.eclipse.californium.core.network.Exchange.Origin;
 import org.eclipse.californium.core.network.config.NetworkConfig;
 import org.eclipse.californium.core.server.MessageDeliverer;
 import org.eclipse.californium.elements.Connector;
 
-
 /**
  * The CoAPStack builds up the stack of CoAP layers that process the CoAP
  * protocol.
  * <p>
  * The complete process for incoming and outgoing messages is visualized below.
  * The class <code>CoapStack</code> builds up the part between the Stack Top and
  * Bottom.
- * <hr><blockquote><pre>
- * +-----------------------+
- * | {@link MessageDeliverer}      |
- * +-------------A---------+
- *               A
- *             * A
- * +-----------+-A---------+
- * |       CoAPEndpoint    |
- * |           v A         |
- * |           v A         |
- * | +---------v-+-------+ |
- * | | Stack Top         | |
- * | +-------------------+ |
+ * <hr>
+ * <blockquote>
+ * 
+ * <pre>
+ * +----------------------------------+
```

Comment:
You probably had good intentions when re-aligning the ASCII graphics. However, I think the result will not look like you expected it to do. The additional *whitespace* characters in lines containing a `{@link}` tag are there in order to make up for the `{@link }` characters which are not visible in the produced HTML ...